Test/TextAnalizerTest.php

```diff
@@ -0,0 +1,96 @@
+<?php
+    namespace test;
+
+    use app\TextAnalizer;
+    use PHPUnit\Framework\TestCase;
+
+    class TextAnalizerTest extends TestCase {
+        
```

Comment:
Just for the your personal development, see the useful link https://phpunit.de/manual/3.7/en/appendixes.annotations.html#appendixes.annotations.dataProvider